### ISSUE TITLE
Code 128 Barcode Type

### ISSUE THREAD

**Issue by nadircan** [ansemjo/angular2-pdf417-barcode#2]
How can i add code 128 barcode type to this package?

**Comment by ansemjo** [comment]
Code 128 is not in the scope of this package, since I am only wrapping [pdf417-js](https://github.com/bkuzmic/pdf417-js) here.

I suggest you use something like [code-128-encoder](https://www.npmjs.com/package/code-128-encoder) and wrap it into a directive or component similar to how I did it here in [index.ts](https://github.com/ansemjo/angular2-pdf417-barcode/blob/master/index.ts). I've stopped using this package myself and haven't switched to Angular 4 yet, so I'm not sure if there are any breaking changes that have to be considered.

Basically you should have a component template with a single `div` of a given class, apply the [necessary styles](https://www.npmjs.com/package/code-128-encoder#use-in-your-browser) and adapt the `update()` function to use `Code128Generator().encode(text)` and update the barcode on changes at the `Input()` binding.

**Comment by ThenmozhiD** [comment]
Whether you have the same for React projects ?

**Comment by ansemjo** [comment]
No I don't and I've never even used React.
